content/04.results.md

```diff
@@ -1,7 +1,42 @@
 # Results
 
-## Random Sampling of Generative Model
-place the grid aurocs here for generative model
+## Generative Model Using Randomly Sampled Label Functions
+
+To evaluate the feasibility of label function reuse we used a random sampling method.
+Our baseline model consisted of training the generative model with only distant supervision label functions for each edge type.
+We report the results in the form of area under the precision recall curve (AUPRC) (Figure {@fig:gen_model_auprc}) and area under the receiver operating curve (AUROC) (Figure {@fig:gen_model_auroc}).  
```

Comment:
See https://github.com/greenelab/deep-review/issues/613... if you want consistency you can do:

+ AUROC and AUPR
+ AUROCC and AUPR